### DISCUSSION TITLE
Can we display block in a modal instead of block manager? If yes, how?

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#3817] [Q&A]
Hi @artf 

    Can we display block in a modal instead of block manager? If yes, how?

Regards

**Comment by artf** [comment]
The modal can receive any HTMLElement as `content`, so if you need, you can pass the block manager element.

**Reply by codingmachine16** [reply]
Sorry, @artf My question was different . I could not explain it. 

Can i display any block manager block icon on any my custom div instead of inside block manager? if yes how?

![image](https://user-images.githubusercontent.com/87995581/136237699-2945cc4a-71df-4054-9975-e403812f0a62.png)

**Reply by Ju99ernaut** [reply]
This should work https://grapesjs.com/docs/api/block_manager.html#render:
```js
// You can also render your blocks outside of the main block container
const newBlocksEl = blockManager.render(filtered, { external: true });
document.getElementById('some-id').appendChild(newBlocksEl);
```
